src/coreclr/jit/gentree.cpp

```diff
@@ -11942,7 +11937,11 @@ void Compiler::gtDispTree(GenTree*     tree,
 
         if (tree->OperIs(GT_FIELD, GT_FIELD_ADDR))
         {
-            printf(" %s", eeGetFieldName(tree->AsField()->gtFldHnd), 0);
+            auto disp = [&]() {
+                char buffer[256];
+                printf(" %s", eeGetFieldName(tree->AsField()->gtFldHnd, true, buffer, sizeof(buffer)));
+            };
+            disp();
```

Comment:
Just avoiding bloating the stack frame here, since `gtDispTree` is recursive (well, it is left up to the C++ compiler's inlining).